.travis.yml

```diff
@@ -22,7 +22,8 @@ addons:
 
 env:
   global:
-  - VERSION="12.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0" UNIT_TEST="0"
+  - OPTIONS="--log-db=openerp_test"
+  - VERSION="master" TESTS="0" LINT_CHECK="0" TRANSIFEX="0" UNIT_TEST="0"
```

Comment:
Why master?

Reply:
It's my fault. I resolved merge conflicts with mistake